program/rust/src/accounts/score.rs

```diff
@@ -0,0 +1,188 @@
+use {
+    super::{
+        AccountHeader,
+        PythAccount,
+    },
+    crate::c_oracle_header::{
+        PC_ACCTYPE_SCORE,
+        PC_MAX_PUBLISHERS,
+        PC_MAX_SYMBOLS,
+        PC_MAX_SYMBOLS_64,
+    },
+    bytemuck::{
+        Pod,
+        Zeroable,
+    },
+    solana_program::{
+        program_error::ProgramError,
+        pubkey::Pubkey,
+    },
+    std::cmp::max,
+};
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+/*
+ * This account is part of Community Integrity Pool (CIP) project.
+ * It is used to store the caps of the publishers which will be sent
+ * to the `integrity_pool` program on mainnet to calculate the rewards.
+ */
```

Comment:
we kind of use `///` in rust everywhere.

Reply:
Fixed.